File: aula02/q4.py
import math

print("Digite Base e Altura do Retângulo: ")
base = int(input())
altura = int(input())

area = base * altura
perimetro = 2 * base + 2 * altura
diagonal = math.sqrt(base ** 2 + altura ** 2)

print(f"Área = {area:.2f} - Perímetro = {perimetro:.2f} - Diagonal = {diagonal:.2f}")

File: Exercicios/lista02_Python_C++/q4.cpp
#include <iostream>
#include <math.h>
using namespace std;

int main(){
    int height, base;
    float area, perimeter, diagonally;

    cout << "Digite um Valor para a Altura e Base de um Retângulo: ";
    cin >> height >> base;

    area = height * base;
    perimeter = height * 2 + base * 2;
    diagonally = sqrt(pow(height, 2) + pow(base, 2));

    cout << "Area = " << area << " - Perímetro = " << perimeter << " - Diagonal = " << diagonally << endl;
    return 0;
}